src/Libraries/RevitNodes/GeometryConversion/RevitToProtoSolid.cs

```diff
@@ -28,10 +28,22 @@ public static class RevitToProtoSolid
             {
                 srfs.AddRange(face.ToProtoType(false));
             }
-            var converted = Solid.ByJoinedSurfaces(srfs);
+
+            Solid converted = null;
+            try
+            {
+                converted = Solid.ByJoinedSurfaces(srfs);
+            }
+            catch (Exception e)
```

Comment:
Just in case Solid.ByJoinedSurfaces() is returning null but does not throw an exception, the check for converted is still reserved. But I have changed to return null if there is an exception.